ega-data-api-it/src/main/java/eu/elixir/ega/ebi/it/dataedge/DataedgeBase.java

```diff
@@ -38,6 +38,7 @@ public DataedgeBase() {
         unencryptedChecksum = System.getProperty("res.file.checksum");
         fileId = System.getProperty("fileId");
         datasetId = System.getProperty("datasetId");
+        RestAssured.useRelaxedHTTPSValidation();
```

Comment:
Why relaxed mode is used here? Can we go for real certs validation?

Reply:
Done.